unit-test.md

```diff
@@ -0,0 +1,19 @@
+Running unit tests
+==============
+
+To run the  unit tests, run the `run-tests.sh` script.  You will need to set these environment variables:
+
+- `SPARK_HOME`: your local copy of Apache Spark. Look at download_spark_dependencies.sh for more details.
```

Comment:
you did a really good job separating the new `spark` module from the rest (for current users). For testing I'd really like if we could have an easy way for contributors to ignore or deactivate Spark tests. Why? Most hyperopt users won't use the new Spark functionality right away and I don't want to discourage new contributions. Plus, I don't think we'll get many PRs for the `spark` module.